--- gnurobbo-0.66/Makefile.libm	2010-08-24 00:41:09.000000000 +0400
+++ gnurobbo-0.66/Makefile	2012-10-22 23:30:29.087060712 +0400
@@ -39,7 +39,7 @@ SDL_CONFIG?=sdl-config
 CFLAGS?=-O3 -pipe
 CFLAGS+=-Wall `$(SDL_CONFIG) --cflags` -DPLATFORM_PC -DVERSION=\"$(VERSION)\" \
 	-DPACKAGE_DATA_DIR=\"$(PACKAGE_DATA_DIR)\" $(FONT_USE_PIXMAP) $(MUSIC) $(DESIGNER)
-LDFLAGS=
+LDFLAGS=-lm
 # Use these instead for debugging and/or profiling (mainly intended for GNU Robbo developers)
 #CFLAGS?=-O0 -pipe -g -pg
 #CFLAGS+=-Wall `$(SDL_CONFIG) --cflags` -DPLATFORM_PC -DVERSION=\"$(VERSION)\" \
